ansible/roles/influxdb/templates/influxdefault

```diff
@@ -0,0 +1 @@
+INFLUXD_OPTS="-join <IP1>:8091"
```

Comment:
In theory this should work: {{ansible_default_ipv4.address|default(ansible_all_ipv4_addresses[0])}}

Can you try testing this through, and see if the IP address if fulled in? <IP1> would be replaced with the item between the braces.

Alterantively "{{ ansible_default_ipv4.address }}" is supposed to work in newer versions of ansible.